android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUIButton.java

```diff
@@ -60,23 +59,23 @@ public TiUIButton(final TiViewProxy proxy)
 		styleId = TiConvert.toInt(proxy.getProperty(TiC.PROPERTY_STYLE), styleId);
 		switch (styleId) {
 			case UIModule.BUTTON_STYLE_OPTION_POSITIVE:
-				styleId = R.attr.buttonBarPositiveButtonStyle;
+				styleId = com.google.android.material.R.attr.buttonBarPositiveButtonStyle;
```

Comment:
To keep the diff small, you could also just `import com.google.android.material.R;` I think.